composer.json

```diff
@@ -31,7 +31,7 @@
     "require-dev": {
         "symfony/yaml": "~2.8 || ~3.4 || ~4.2 || ~5.0",
         "phpunit/php-code-coverage": "^2.2|^3.0",
```

Comment:
Is this dependency required here? Does not PHPUnit rerquire it itself?

Reply:
Looks like it, so I would agree it can probably be removed

4.8.36 has phpunit/php-code-coverage: ~2.1
5.7.27 has phpunit/php-code-coverage: ^4.0.4

etc